java/tools/src/test/java/org/apache/arrow/tools/TestIntegration.java

```diff
@@ -274,8 +274,7 @@ public void testInvalid() throws Exception {
       integration.run(args3);
       fail("should have failed");
     } catch (IllegalArgumentException e) {
-      assertTrue(e.getMessage(), e.getMessage().contains("Different values in column"));
-      assertTrue(e.getMessage(), e.getMessage().contains("999"));
+      assertTrue(e.getMessage(), e.getMessage().contains("Different vectorSchemaRoots"));
```

Comment:
Can you explain this change? I don't the the exception message should be changing

Reply:
Before this change, Validator could throw Exception with different message, since we make it return boolean value, log the explicit message instead of throwing it.